lib/knapsack/presenter.rb

```diff
@@ -51,10 +51,15 @@ def time_offset_warning
         }
         if Knapsack.tracker.time_exceeded?
           str << %{
-Tests on this CI node took more than max allowed node time execution.
+Tests on this CI node took more than the max allowed node time execution.
 Please regenerate your knapsack report.
-If that didn't help then split your heavy test file
-or bump time_offset_in_seconds setting.}
+
+If that didn't help then split your slow test file into smaller test files
+or bump time_offset_in_seconds setting.
+
+You can also use knapsack_pro gem to automatically divide slow test files between parallel CI nodes.
```

Comment:
How about:
`You can also allow the knapsack_pro gem to automatically divide your slow test files across parallel CI nodes.`